static_src/routes.js

```diff
@@ -93,7 +93,8 @@ export function space(orgGuid, spaceGuid, next) {
   spaceActions.fetch(spaceGuid);
   serviceActions.fetchAllInstances(spaceGuid);
   userActions.changeCurrentlyViewedType('space_users');
-  userActions.fetchSpaceUsers(spaceGuid);
+  userActions.fetchOrgUsers(orgGuid);
```

Comment:
So I found that there's no CF API equivalent to `fetchSpaceUsers` like `fetchOrgUsers`. So essentially, you have to fetch org users and then fetch space user roles to assign all the users to specific roles for a space.